news/2024/2024-09-14-new-featured-artist-tomspicy.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "New Featured Artist: tomspicy"
+date: 2024-09-14 19:00:00 +0000
+---
+
+Send only your warmest welcomes to **tomspicy**, our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/437/header.jpg?1)
+
+**tomspicy** is an electronic artist whose music career was kickstarted by passionate public reception to a meme track: the [fieryrage accordion](https://www.youtube.com/watch?v=yoV794KYq48) [hardcore remix](https://osu.ppy.sh/beatmapsets?q=%22fieryrage%20accordion%22)".
+
+Ever since the short tune's release in 2021, **tomspicy** has been dedicated to electronic music, primarily outputting drum and bass with occassionally dips into other electronic subgenres like hardcore and old school jungle. As of today, **tomspicy** is also dipping into our Featured Artist library!
+
+**16** tunes available on [**tomspicy**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/437), including an extended version of the fieryrage accordion hardcore remix and three new [original songs](/wiki/osu!_originals) produced specifically for this Featured Artist reveal!
+
+For who haven't heard **tomspicy**'s sound yet, your first experience can be through [jackylam5](https://osu.ppy.sh/users/1540807) unfolding a new way to play osu! from [COE 2024](/wiki/Community/cavoe's_osu!_event/2024):
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/437/release_showcase.mp4?1" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### tomspicy - fieryrage accordion (hardcore remix) (TV Size)
+
+Take your pick:
+
+- [The map from the video above](https://osu.ppy.sh/beatmapsets/1516867) hosted by [Aakki](https://osu.ppy.sh/users/11077540)
+- [This osu!taiko rendition](https://osu.ppy.sh/beatmapsets/1516860) hosted by [Hivie](https://osu.ppy.sh/users/14102976)
+- [An osu!catch difficulty spread](https://osu.ppy.sh/beatmapsets/1921382) hosted by [Juansy-](https://osu.ppy.sh/users/12588215)
+- [Your own future osu!mania map](https://assets.ppy.sh/artists/437/tomspicy%20osu!%20collection%2001/tomspicy%20-%20fieryrage%20accordion%20(hardcore%20remix)%20(TV%20Size).osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/437/tomspicy%20osu!%20collection%2001/tomspicy%20-%20fieryrage%20accordion%20(hardcore%20remix)%20(TV%20Size).mp3">
+</audio>
+
+### tomspicy - dysthymia
```

Comment:
fa listing says that the title is (sic) "dsythymia", might want to cross-check this one (or fix the one in the listing if that one was the mistake)

![image](https://github.com/user-attachments/assets/b31993f5-855e-44b0-90cd-b0bdd622e21e)

Reply:
fixed